model/scratch/setup.py

```diff
@@ -0,0 +1,58 @@
+import os
+import ast
+from io import open
+
+from setuptools import find_packages, setup
+
+self_path = os.path.dirname(os.path.realpath(__file__))
+
+with open(os.path.join(self_path, 'dffml_model_scratch', 'version.py'),
+          'r') as f:
+    for line in f:
+        if line.startswith('VERSION'):
+            version = ast.literal_eval(line.strip().split('=')[-1].strip())
+            break
+
+with open(os.path.join(self_path, 'README.md'), 'r', encoding='utf-8') as f:
+    readme = f.read()
+
+INSTALL_REQUIRES = [
+    ]
+
+setup(
+    name='dffml-model-scratch',
+    version=version,
+    description='',
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    author='John Andersen',
+    author_email='john.s.andersen@intel.com',
```

Comment:
You're the author of this one :)

Reply:
Oops! Forgot to change that...

Reply:
Add two more lines here:

```python
    maintainer='John Andersen',
    maintainer_email='john.s.andersen@intel.com',
```

Since you authored the package, but I'll be the one managing update to the package repository